docs/layout.html

```diff
@@ -38,5 +39,62 @@ <h1><img src="logo.svg"> Mithril <small>[version]</small></h1>
 			}
 		}
 		</script>
+		<script src="https://flems.io/flems.html" defer></script>
+		<style type="text/css" media="screen">
+			.flems main {
+				margin: 0;
+				max-width: auto;
+			}
+			.flems {
+				margin: 20px;
+				max-height: 400px;
+				box-shadow: 0 0 20px rgba(0,0,0,0.25);
+			}
+			@media (max-width: 500px) {
+				.flems {
+					min-height: calc(100vw * 1.3);
+				}
+			}
+		</style>
+		<script>
```

Comment:
Could you move this into the same `<script>` as the hamburger menu code?

Also, it might be better to use `<script src="https://flems.io/flems.html?nextie2" async id="flems">` and `document.getElementById("flems").onload = ...`, so the browser can fetch it a little quicker.